core/rawdb/freezer.go

```diff
@@ -325,23 +325,28 @@ func (f *Freezer) validate() error {
 		return nil
 	}
 	var (
-		length uint64
-		name   string
+		head uint64
+		tail uint64
+		name string
 	)
 	// Hack to get length of any table
 	for kind, table := range f.tables {
-		length = atomic.LoadUint64(&table.items)
+		head = atomic.LoadUint64(&table.items)
+		tail = atomic.LoadUint64(&table.itemHidden)
 		name = kind
 		break
 	}
 	// Now check every table against that length
```

Comment:
maybe update the comment to say "against those boundaries" since the length variable is no longer used?

Reply:
fixed